src/socket-io.js

```diff
@@ -11,7 +11,7 @@ import { createEvent, createMessageEvent, createCloseEvent } from './event/facto
  *
  * http://socket.io/docs/
  */
-class SocketIO extends EventTarget {
+export class SocketIO extends EventTarget {
```

Comment:
this class is already exported as default

Reply:
The default export is the `IO` function, which returns a new SocketIO instance:

https://github.com/thoov/mock-socket/blob/f1d5de4e6409742670303b6666d833cb4f3af033/src/socket-io.js#L280-L282

The exported `SocketIO` class is used in an `instanceof` check. We can't use the `IO` export for that, right?

Reply:
Ah, I see. So it's ok, thanks